src/components/forms/OpaqueIngressSecretForm.js

```diff
@@ -186,12 +223,15 @@ class OpaqueIngressSecretForm extends Component {
                 value={this.state.arbitrary_secret_name}
                 onChange={this.handleInputChange}
                 invalidCallback={() =>
-                  this.props.secretCallback(this.state, this.props)
+                  this.props.secretCallback("arbitrary_secret_name")(
```

Comment:
we should not be hardcoding `"arbitrary_secret_name ` here, this callback should only take two arguments, `stateData` and `componentProps`